### DISCUSSION TITLE
Magit satus is very slow compared to my previous experience.

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#91] [Q&A] [answered]
Hi. 

I just started using MinEmacs yesterday. Previously I use Doom. One thing that bugs me the most is that `magit-status` is very slow. So when pressing `SPC g g` you can see `Turning on magit-auto-revert-mode...done` takes 2-3 before showing the git log. It happens even for tiny repo with few commits.

Do you notice this behavior too? Or is it only on my end?
How to track the time invoked for a command, so I can precisely compare between configs.

Thanks! :heart:

**Comment by abougouffa** [comment, marked as answer]
This happens only on the first time you run `SPC g g` after opening a new Emacs instance. MinEmacs prefers deferring all packages to save startup time. Pressing `SPC g g` for the first time will load `magit` and other packages like `forge` and `code-review`.

The second time you press `SPC g g` it should start smoothly.

**Reply by azzamsa** [reply]
Confirmed!. Thanks! :heart:
